### PR TITLE
Menubar and welcome window

### PR DESCRIPTION
**Pull request by pemba1s1**
### Screenshots
image
image
image

## Summary by Sourcery

Add a menu bar with Settings and Help options to the main window and introduce a welcome message feature. Enhance the settings window with a general settings tab and improve UI by rendering markdown content in help and welcome messages. Update documentation with new markdown files for welcome and about sections.

New Features:
- Introduce a menu bar with Settings and Help options in the main window.
- Add a welcome message that displays upon application launch, configurable via settings.

Enhancements:
- Refactor settings window to include a general settings tab.
- Improve the UI by adding HTML rendering for markdown content in help and welcome messages.

Documentation:
- Add a welcome markdown file to introduce the AI Medical Scribe project and its features.
- Include an about markdown file to provide information on the AI-Scribe project.

### CODE REVIEW

File: src/FreeScribe.client/UI/MainWindowUI.py [hunk 1]
```diff
@@ -118,6 +132,84 @@ def destroy_docker_status_bar(self):
...
+            checkbox = tk.Checkbutton(help_window, text="Don't show this message again", variable=dont_show_again)
+            checkbox.pack(side=tk.BOTTOM, pady=10)
+            # Update the setting when the window is closed
+            help_window.protocol("WM_DELETE_WINDOW", lambda: self._on_help_window_close(help_window, dont_show_again))
+
+        # Ensure the help window is always on top
+        help_window.lift()
+
+        # Set the size of the help window to height and width of 900px
+        help_window.geometry(f"900x900")
```
[the remark this answers is not in the document]
Comment (pemba1s1) [reply src/FreeScribe.client/UI/MainWindowUI.py:193]: window is resizable

File: src/FreeScribe.client/UI/MainWindowUI.py [hunk 2]
```diff
@@ -118,6 +132,84 @@ def destroy_docker_status_bar(self):
...
+
+    def _create_settings_menu(self):
+        # Add Settings menu
+        self.menu_bar.add_command(label="Settings", command=self.setting_window.open_settings_window)
+
+    def _create_help_menu(self):
+        # Add Help menu
+        help_menu = tk.Menu(self.menu_bar, tearoff=0)
+        self.menu_bar.add_cascade(label="Help", menu=help_menu)
+        help_menu.add_command(label="About", command=lambda: self._show_md_content('src/FreeScribe.client/markdown/help/about.md', 'About'))
```
[the remark this answers is not in the document]
Comment (yingbull) [reply src/FreeScribe.client/UI/MainWindowUI.py]: This is fair.

File: src/FreeScribe.client/UI/MainWindowUI.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (yingbull) [reply src/FreeScribe.client/UI/MainWindowUI.py]: Fair to consider it with screen geometry though, but do not need to change right now.

[the remark this answers is not in the document]
Comment (yingbull) [reply src/FreeScribe.client/UI/MainWindowUI.py]: Fair.

File: src/FreeScribe.client/markdown/welcome.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Welcome to the AI Medical Scribe Project
+
+Welcome to the AI Medical Scribe project! This project aims to provide an intelligent medical scribe application that assists healthcare professionals by transcribing conversations and generating medical notes.
+
+## Introduction
+
+The AI Medical Scribe project leverages advanced machine learning models to transcribe conversations between healthcare providers and patients. It also generates structured medical notes based on the transcriptions, helping to streamline the documentation process in clinical settings.
+
+## Features
+
```
[the remark this answers is not in the document]
Comment (yingbull) [reply src/FreeScribe.client/markdown/welcome.md]: OK for now.

**Review by yingbull — CHANGES_REQUESTED** [review]

**Comment by pemba1s1** [comment]
@yingbull I have already made all the changes requested by the bot earlier. Forgot to comment it.

**Review by yingbull — APPROVED** [review]

**Pull request merged**